src/components/util/util.test.js

```diff
@@ -246,3 +247,31 @@ describe('hasEveryLoanItemStatus', () => {
     expect(hasEveryLoanItemStatus(loans, status)).toBe(true);
   });
 });
+
+describe('getContributors', () => {
+  it('should return undefined if nothing is passed', () => {
+    expect(getContributors()).toBe();
+  });
+
+  it('should return contributors from `account` if it passed', () => {
+    const mockedAccount = {
+      contributors: [{ name: 'Bond,James' }, { name: 'Doe,John' }],
+    };
+    const mockedInstance = {
+      contributors: [{ name: 'test' }],
+    };
+    const expectedResult = ['James, Bond', 'John, Doe'];
```

Comment:
Is `James, Bond` really the format we want here? I would expect `James Bond`. Maybe the final `join(', ')` should just be `join(' ')`?

Reply:
The logic for concatenating parts of strings is just copied. But if you think so, we can discuss it with PO (should we?). Is it worth changing it as part of the current ticket or maybe as part of a separate one? Thanks.

Reply:
@zburke, can we resolve this conversation?

Reply:
No, in my opinion, we cannot resolve this conversation because I think this formatter is wrong. 

Yes, please discuss it with your PO, or LMK who to contact if you would like me to do that.

Reply:
Asked this in the comments: https://issues.folio.org/browse/UIU-2092

Reply:
Should not be changed in accordance with PO answer https://issues.folio.org/browse/UIU-2092?focusedCommentId=111852&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-111852